ocpp-json/src/main/scala/com/thenewmotion/ocpp/json/v16/SerializationV16.scala

```diff
@@ -478,6 +488,15 @@ object SerializationV16 extends SerializationCommon {
     )
   }
 
+  private implicit class RichDataTransferStatus(dataTransferStatus: messages.DataTransferStatus) {
+    def toV16: String = dataTransferStatus.name
+  }
+
+  private implicit class RichV16DataTransferStatus(self: String) {
+    def fromV16: messages.DataTransferStatus = messages.DataTransferStatus.withName(self).get
```

Comment:
Aren't you calling .get here on an `Option` that is created from untrusted network input?

Reply:
or well, typically created from untrusted network input, in the end it's just whatever people end up calling the library on. Still I'd say there are better ways to handle this, and there is already an `enumerableFromJsonString` function in use in this file to deal with exactly this type of conversion.